components/scream/cmake/SetCompilerFlags.cmake

```diff
@@ -92,13 +92,24 @@ if (${CUDA_FOUND})
   # We found cuda, but we may be only interested in running on host.
   # Check if the compiler is not nvcc; if not, do not add cuda support
   execute_process(COMMAND ${CMAKE_CXX_COMPILER} "--nvcc-wrapper-show"
-                  RESULT_VARIABLE WRAPS_NVCC
-                  OUTPUT_VARIABLE WRAPS_NVCC_OUT
-                  OUTPUT_QUIET
-                  ERROR_QUIET)
-  string (FIND "${WRAPS_NVCC_OUT}" "nvcc" pos)
+    RESULT_VARIABLE WRAPS_NVCC
```

Comment:
@bartgol , I know you pulled this code from somewhere else (hommexx?), so I wanted to let you know that it had bugs. Setting OUTPUT_QUIET means the command produces no output, so cuda was never being found. I also had to make changes to support the case where the user is using mpi and OMPI_CXX.

Reply:
Ah, good catch. I think this was just for scream, I don't know if we have it in homme too. But I guess we should, cause with internal kokkos we should still have this problem there too. Thanks for the fix.